services/api.py

```diff
@@ -861,8 +864,17 @@ def service_nodes_by_ancestors(service_node_ids):
 
         if 'observations' in self.include_fields:
             queryset = queryset.prefetch_related(
-                'observation_set__property__allowed_values').prefetch_related(
```

Comment:
The prefetch_related for ```observation_set__property__allowed_values``` has been removed. Intentionally?

Reply:
Added back observation_set__property__allowed_values.